data/voxforge/run_data.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+cd ../.. > /dev/null
+
+# download data, generate manifests
+PYTHONPATH=.:$PYTHONPATH python data/voxforge/voxforge.py \
+--manifest_prefix='data/voxforge/manifest' \
```

Comment:
Now that run the script in the same directory, ```cd ../.. > /dev/null``` and the prefix ```data/voxforge``` won't be necessary.

Reply:
Done.